app/models/actor.rb

```diff
@@ -0,0 +1,12 @@
+class Actor < ApplicationRecord
+  has_many :actor_movies
+  has_many :movies, through: :actor_movies
+
+  def self.order_by_age
+    order(age: :desc)
```

Comment:
You're on the right track here by encapsulating this logic! We want these queries to live in a model method, but in this case we want this logic to be for a *given movie*, so we'd want to call this from a movie instance in the Movie model like so:
`self.actors.order(:age)` where self is the movie instance. 

Same with average age-- the US is looking for the avg age of actors for this given movie rather than all actors in the database